### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by Faancy**
Related issues: [please specify]

## Description: A picture and a security picture

What are you up to? Fill us in :)

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: content/tech-big-picture/security-best-practices-for-you/_index.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ prerequisites:
 ready: true
 title: Security best practices for you
 ---
+![best security practices](security.png)
```
Comment (sheenarbw) [inline content/tech-big-picture/security-best-practices-for-you/_index.md:10]: where is security.png? I'm not seeing it in the repo so this picture will be broken. Did you forget to add it to your PR?

File: content/tech-big-picture/security-best-practices-for-you/_index.md [hunk 2]
```diff
@@ -25,5 +26,5 @@ These practices are important because they significantly reduce the risk of a se
 
 ## Resources 
 
-- https://www.nytimes.com/guides/privacy-project/how-to-protect-your-digital-privacy
-- https://www.zdnet.com/article/online-security-101-how-to-protect-your-privacy-from-hackers-spies-and-the-government/
\ No newline at end of file
+- (How to protect your digital privacy)[https://www.nytimes.com/guides/privacy-project/how-to-protect-your-digital-privacy]
+- 9how to protect your privacy from hackers, spies, and the government)[https://www.zdnet.com/article/online-security-101-how-to-protect-your-privacy-from-hackers-spies-and-the-government/]
```
Comment (sheenarbw) [inline content/tech-big-picture/security-best-practices-for-you/_index.md]: Typo here. `9how `should be `(how`

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This one looks a bit rushed. Please always take a minute to check that you would approve the PR

File: content/tech-big-picture/security-best-practices-for-you/_index.md (hunk 1 above)
Comment (Faancy) [reply content/tech-big-picture/security-best-practices-for-you/_index.md:10]: I just added it to the branch.

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**